calcom/cal.com
# Update calendar when new attendees are added to bookings w/ seats

## What does this PR do?



This PR updates the organizer's calendar when new attendees are added to booking's with seats.

While working through this I noticed there are some improvements I can make to the seats feature so I'll be working on those.

Fixes # (issue)



**Environment**: Staging(main branch) / Production

## Type of change





## How should this be tested?



- Create an event type with seats
- Book a slot
- Now book that same slot with a different attendee
- Your connected calendar should show both attendees

## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
